projects/dex-ui/src/pages/Well.tsx

```diff
@@ -19,6 +22,13 @@ export const Well = () => {
       <strong>Tokens: {well.tokens!.map((t) => t.symbol).join(":")}</strong>
       <br />
       <strong>Reserves: {well.reserves!.map((r) => r.toHuman()).join(":")}</strong>
+      <br />
+      <div>
+        <h1>Swaps history</h1>
+        {swaps.map((swap) => (
```

Comment:
doesn't this trigger a ts error? that `swaps` might be undefined until after it loads successfully? might need to be `swaps && swaps.map()..` or something

Reply:
Though I realize I need to add a check for `error` in the `useEffect` hook.